src/store/channels-list/channels.ts

```diff
@@ -0,0 +1,10 @@
+import { multicastChannel } from 'redux-saga';
+import { call } from 'redux-saga/effects';
+
+let theConversationsChannel;
+export function* conversationsChannel() {
+  if (!theConversationsChannel) {
+    theConversationsChannel = yield call(multicastChannel);
```

Comment:
what does calling `multicastChannel` do?

Reply:
`multicastChannel` is a redux-saga thing. It allows you to publish an event that multiple listeners can "take" from.